### ISSUE TITLE
About the source code for 'teacher network'

### ISSUE THREAD

**Issue by chgao96** [facebookresearch/DensePose#53]
It is mentioned in the paper that there is a so-called teacher network to reconstruct the ground-truth values. I tried my best to find the code for teacher network in this published repo but with no results. So hope for a response. Thanks a lot!

**Comment by nvrv** [comment]
@chgao96 This DensePose-RCNN implementation is an updated version of what was presented in the paper and it does not require the dense ground truth for training. For this reason, the teacher network is not a part of this repository. Instead, all provided models were trained directly on collected human annotations and achieve similar performance.

**Comment by chgao96** [comment]
Thanks for your reply. :-) So there are some differences between this implementation and the published paper. Then is there a document or tutorial describing those differences? It will help me in studying both the code and the paper.

**Comment by zhengzl** [comment]
Thanks for your reply. Could you recommend some source code for training teacher network, tuning hyper-parameters and improving efficiency of the implementation? I really appreciate for your help for retraining DensePose model.

**Comment by zhengzl** [comment]
@nvrv Thanks for your reply. Could you recommend some source code for training teacher network, tuning hyper-parameters and improving efficiency of the implementation? I really appreciate for your help for retraining DensePose model.

**Comment by GianKiMoon** [comment]
How exactly is the supervision signal constructed when you don't use the teacher network? When you use the inpainting you get basically a dense mask, so I get this point, but when I use the sparse points how do I get a supervision signal for a FCN output? Thanks!
